diff --git a/Gruntfile.js b/Gruntfile.js
index 98034bf..afeeac9 100644
--- a/Gruntfile.js
+++ b/Gruntfile.js
@@ -14,8 +14,7 @@ module.exports = function(grunt) {
 			},
 			banner: '<%= grunt.template.today("yyyy-mm-dd") + "\\n" %>' +
 				'<%= pkg.homepage ? "* " + pkg.homepage + "\\n" : "" %>' +
-				'* Copyright (c) <%= grunt.template.today("yyyy") %> <%= pkg.author.name %>;' +
-				' Licensed <%= _.pluck(pkg.licenses, "type").join(", ") %> */\n\n'
+				'* Copyright (c) <%= grunt.template.today("yyyy") %> <%= pkg.author.name %> */\n\n'
 		},
 
 		sass: {
